home.html

```diff
@@ -77,5 +77,152 @@ <h2 class="profession-title">I'm a Software Developer</h2>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
+          <a href="#" class="btn btn-solid-long">See Project</a>
+        </div>
+        <div class="work-item">
+          <h3 class="sub-heading">Profesional Art Printing Data</h3>
+          <p>
+            A daily selection of privately personalized reads; no accounts or
+            sign-ups required. has been the industry's standard
+          </p>
+          <div class="button-group-2">
+            <a href="#" class="btn btn-gray">HTML</a>
+            <a href="#" class="btn btn-gray">Bootstrap</a>
+            <a href="#" class="btn btn-gray">Ruby</a>
+          </div>
```

Comment:
I think the list of project technologies (e.g., "HTML", "Bootstrap" and "Ruby") should be a list `ul` and each of them a `li`. Please add the `ul` tag instead of the `div` tag and replace the `a` tag with a `li` tag 😊 .